src/js/controller/controller.js

```diff
@@ -773,19 +773,27 @@ Object.assign(Controller.prototype, {
 
         function _seek(pos, meta) {
             const state = _model.get('state');
+            const activeEl = document.activeElement;
+
             if (state === STATE_ERROR) {
                 return;
             }
             if (meta && state === STATE_PLAYING && _model.get('playReason') !== meta.reason) {
                 _model.set('playReason', meta.reason);
             }
+            if (meta && !activeEl.classList.contains('jw-slider-time')) {
+                _model.set('playReason', 'external');
+            }
             _programController.position = pos;
             const isIdle = state === STATE_IDLE;
             if (!_model.get('scrubbing') && (isIdle || state === STATE_COMPLETE)) {
                 if (isIdle) {
                     meta = meta || {};
                     meta.startTime = pos;
                 }
+                if (meta.reason !== _model.get('playReason')) {
```

Comment:
this is for when the player is in an idle state. When I tried to start playback with the seek method, I kept seeing "interactive".

Reply:
We have `if (isIdle)` block right above this, can we do `meta.reason = 'external';` up there instead of creating a new `if`, unless there was a specific reason for having it?

Reply:
will give it a try and see if this fixes the bug!